app.py

```diff
@@ -31,6 +34,15 @@ def create_app():
     oauth_client = WebApplicationClient(client_id)
     login_manager.init_app(app)
 
+    app.logger.setLevel(os.environ.get('LOG_LEVEL', 'DEBUG'))
+
+    loggly_token = os.environ.get('LOGGLY_TOKEN')
+
+    if loggly_token is not None:
```

Comment:
You could change this to also skip an empty string value:

```suggestion
    if loggly_token:
```